Sources/MapboxCoreNavigation/EHorizon/RoadObjectsStore.swift

```diff
@@ -56,6 +56,36 @@ public final class RoadObjectsStore {
         return native.getRoadObjectIdsByEdgeIds(forEdgeIds: edgeIdentifiers.map(NSNumber.init))
     }
 
+    /**
+     Adds a road object to be tracked in the electronic horizon. In case if an object with such identifier already exists, updates it.
+     NB: road objects obtained from route alerts cannot be added via this API.
+
+     - parameter roadObject: Custom road object, acquired from `RoadObjectMathcer`.
+     */
+    public func addCustomRoadObject(_ roadObject: RoadObject) {
+        guard let nativeObject = roadObject.native else {
+            assertionFailure("You can only add matched custom road objects, acquired from RoadObjectMatcher.")
+            return
+        }
+        native.addCustomRoadObject(for: nativeObject)
+    }
+
+    /**
+     Removes road object and stops tracking it in the electronic horizon.
+
+     - parameter identifier: Identifier of the road object that should be removed.
+     */
+    public func removeCustomRoadObject(identifier: RoadObjectIdentifier) {
+        native.removeCustomRoadObject(forId: identifier)
+    }
+
+    /**
+     Removes all custom road objects from the store and stops tracking them in the electronic horizon.
+     */
+    public func removeAllCustomRoadObjects() {
```

Comment:
Rename to `removeAllRoadObjects()` or, if we really need to emphasize that only application-provided road objects are removed, then `removeAllUserDefinedRoadObjects()`.